internal/provider_count.go

```diff
@@ -0,0 +1,5 @@
+package internal
+
+// This file will be updated automatically, do not edit it manually.
```

Comment:
Can we rename the folder or file to include "generated" to make it little more obvious?